.github/workflows/release.yml

```diff
@@ -0,0 +1,189 @@
+name: "Release"
+
+on:
+  push:
+    tags:
+      - "*"
+
+env:
+  QT_VERSION: 5.15.0
+  VC_REDIST_VERSION: 14.26.28720
```

Comment:
Weekly or Bi-weekly Github Action updates the Windows Image and this number may change if VS has new updates. 
Wouldn't Line 127 cd not fail?